content/about.md

```diff
@@ -0,0 +1,19 @@
++++
+title = "About"
+slug = "about"
++++
+
+This is the website for the [Linebender organization on GitHub]. The Linebender organization contains a collection of repositories, which when taken as a whole provide the tools and utilities required to create applications in Rust with a user interface.
+
+It is also a group of volunteers and enthusiasts who hang out on [our Zulip][xi.zulip], and who collaborate on designing and coding the different parts of the stack required for UI. We are a friendly bunch, and always welcome newcomers, whether they have any experience with 2D graphics and user interfaces or not.
+
+All the work of the Linebender is done in the open, and the full history of all conversations is available on Zulip. In fact, we recommend searching the chat history for any topic that you are interested in, as there have been many discussions in the past with novel and valuable ideas. All code is dual-licenses under the MIT and Apache-2.0 licenses (where possible) and we aim to license non-code material under the [CC-BY Creative Commons license][CC-BY]. In fact, the non-code portion of this website is licensed under CC-BY, while all code, including any JavaScript written by the authors, is available under the Apache-2.0 license. A copy of the Apache-2.0 license is available in [this website's repository][website repository].
```

Comment:
I think this copy of the Apache license isn't present

I think we also need to mention our use of the minima Jekyll theme, which is MIT licensed.
I *think* that only applies to base.html and main.scss.

Reply:
I was 100% sure I'd added the apache license, but lo and behold when I looked it was missing :confused:. I've added it now, and in addition added a short message about the reuse of 'minima' in about.md.